readme.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const ruleMod = require('./rules');
```

Comment:
Similar question as with the `readme.js` file. Do you have a better name or placement for `rules.js`?